Source/DataSources/KmlDataSource.js

```diff
@@ -1230,15 +1230,16 @@ define([
         var extrude = queryBooleanValue(geometryNode, 'extrude', namespaces.kml);
         var tessellate = queryBooleanValue(geometryNode, 'tessellate', namespaces.kml);
         var canExtrude = isExtrudable(altitudeMode, gxAltitudeMode);
-
-        if (defined(queryNumericValue(geometryNode, 'drawOrder', namespaces.gx))) {
-            oneTimeWarning('kml-gx:drawOrder', 'KML - gx:drawOrder is not supported in LineStrings');
-        }
+        var zIndex = queryNumericValue(geometryNode, 'drawOrder', namespaces.gx);
 
         var ellipsoid = dataSource._ellipsoid;
         var coordinates = readCoordinates(coordinatesNode, ellipsoid);
         var polyline = styleEntity.polyline;
         if (canExtrude && extrude) {
+            if (defined(zIndex)) {
```

Comment:
This check isn't needed, Google Earth silently ignores zIndex for extruded geometry and since we match GE/KML, there's no reason for us to warn here either.